examples/solverdummies/precice-config.xml

```diff
@@ -12,12 +12,12 @@
   <data:vector name="Data-One" />
   <data:vector name="Data-Two" />
 
-  <mesh name="SolverOne-Mesh">
+  <mesh name="SolverOne-Mesh" dimensions="3">
     <use-data name="Data-One" />
     <use-data name="Data-Two" />
   </mesh>
 
-  <mesh name="SolverTwo-Mesh">
+  <mesh name="SolverTwo-Mesh" dimensions="3">
```

Comment:
Now that we have "duplicate" dimensions configuration, should we also check that the dimensions are the same? If yes, where? I guess somewhere in each mapping that assumes same dimensions.